<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Produto</class>
 <widget class="QWidget" name="Produto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adaptive</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1" rowspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pushButton_3">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
outline: none;
font: 57 bold 12pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border: 2px solid white;
border-top: 1px solid white;
border-left: 1px solid white;
padding: 5px;
}
QPushButton::hover
{
border: 2px solid rgb(255, 87, 34);
background-color: rgb(255, 87, 34);
}
QPushButton::focus
{
border: 2px solid rgb(255, 87, 34);
background-color: rgb(255, 87, 34);
}

</string>
     </property>
     <property name="text">
      <string>Sair</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_6">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
outline: none;
font: 57 bold 12pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border: 2px solid white;
border-top: 1px solid white;
border-right: 1px solid white;
padding: 5px;
}
QPushButton::hover
{
 border: 2px solid #FECA1E;
background-color: #FECA1E;
}
QPushButton::focus
{
 border: 2px solid #FECA1E;
background-color: #FECA1E;
}</string>
     </property>
     <property name="text">
      <string>Atualizar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
border-color: rgb(3, 54, 104);
border:transparent;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 57 bold 16pt &quot;Ubuntu&quot;;
padding-bottom:10px;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff5722;&quot;&gt;Overview&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section {
                            border: none;
                            background-color: rgb(0, 0, 0);
                            color: rgb(255, 255, 255);
								font: 57 12pt &quot;Ubuntu&quot;;
};
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButton_7">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
outline: none;
font: 57 bold 12pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border: 2px solid white;
border-top: 1px solid white;
border-left: 1px solid white;
border-right: 1px solid white;
padding: 5px;
}
QPushButton::hover
{
border: 2px solid #0D355A;
background-color: #0D355A;
}
QPushButton::focus
{
border: 2px solid #0D355A;
background-color: #0D355A;
}</string>
     </property>
     <property name="text">
      <string>Limpar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
border:transparent;
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="6" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-bottom: 1px solid;
border-bottom-color: rgb(255, 87, 34);
font: 57 11pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="prefix">
         <string>R$ </string>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>294</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 57 16pt &quot;Ubuntu&quot;;
padding-bottom:5px;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff5722;&quot;&gt;Produto&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-bottom: 1px solid;
border-bottom-color: rgb(255, 87, 34);
font: 57 11pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 57 14pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Produto:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="indent">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 57 14pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Preço: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="indent">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBox">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-bottom: 1px solid;
border-bottom-color: rgb(255, 87, 34);
font: 57 11pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 57 14pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Código:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="indent">
         <number>48</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
outline: none;
background-color: rgb(0, 0, 0);
font: 57 9pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border-color: rgb(255, 87, 34);
border-style: solid;
border-width:1px;
border-radius:15px;
max-width:30px;
max-height:30px;
min-width:30px;
min-height:30px;
}

QPushButton::hover
{
border-color: rgb(255, 255, 255);
color: rgb(255, 87, 34);
}
</string>
        </property>
        <property name="text">
         <string>Rand</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>doubleSpinBox</zorder>
     <zorder>lineEdit</zorder>
     <zorder>pushButton</zorder>
     <zorder>spinBox</zorder>
     <zorder>label_2</zorder>
     <zorder>label_5</zorder>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
outline: none;
font: 57 bold 12pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border: 2px solid white;
border-bottom: 1px solid white;
border-right: 1px solid white;
padding: 5px;
}
QPushButton:hover
{
	border: 2px solid #0073e5;
	background-color: #0073e5;
}
QPushButton:focus
{
	border: 2px solid #0073e5;
	background-color: #0073e5;
}
</string>
     </property>
     <property name="text">
      <string>Inserir </string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_5">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
outline: none;
font: 57 bold 12pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border: 2px solid white;
border-bottom: 1px solid white;
border-left: 1px solid white;
border-right: 1px solid white;
padding: 5px;
}
QPushButton::hover
{
border: 2px solid #60061E;
background-color: #60061E;
}
QPushButton::focus
{
border: 2px solid #60061E;
background-color: #60061E;
}</string>
     </property>
     <property name="text">
      <string>Deletar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButton_4">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
outline: none;
font: 57 bold 12pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border: 2px solid white;
border-bottom: 1px solid white;
border-left: 1px solid white;
padding: 5px;
}
QPushButton::hover
{
border: 2px solid #1E2A2D;
background-color: #1E2A2D;
}
QPushButton::focus
{
border: 2px solid #1E2A2D;
background-color: #1E2A2D;
}</string>
     </property>
     <property name="text">
      <string>About Qt</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>spinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
